register.lic

```diff
@@ -0,0 +1,76 @@
+=begin
+  Search a deed register for the specified crafting material.
+  Output all lines that match.
+=end
+
+custom_require.call(%w[common common-items drinfomon])
```

Comment:
What is `drinfomon` used for?

Reply:
Not sure.  I used scroll-search.lic to learn how to do some of the things I wasn't sure of, and it was used in that.  I'll take it out and see if it still works.